app/models/manageiq/providers/vmware/infra_manager/refresh_worker/runner.rb

```diff
@@ -27,6 +27,17 @@ def do_work
     super
   end
 
+  def deliver_queue_message(msg)
+    return super unless ems.supports_streaming_refresh?
+
+    msg.delivered_on
```

Comment:
Does this do anything?

Reply:
Yeah it prevents the miq_queue log like from saying the msg finished before it started